tests/range/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1,12 +1,77 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Can you add tests for more than a single statement?

```js
call(
  1, 2,3
);
[cursor[
call(
  1, 2,3
);

call(
  1, 2,3
);
]cursor]
call(
  1, 2,3
);
```


```js
call(
  1, 2,3
);

call(
  1, [cursor[2,3
);

call(
  1, ]cursor]2,3
);

call(
  1, 2,3
);
```

those two things should format the middle two only.